experimenter/experimenter/jetstream/tasks.py

```diff
@@ -47,13 +52,16 @@ def fetch_jetstream_data():
                     >= dt.date.today()
                 )
             ):
-                logger.info(f"Fetching Jetstream data for {experiment.name}")
+                logger.info(
+                    f"Fetching Jetstream data for {experiment.name} ({experiment.slug})"
+                )
                 fetch_experiment_data.delay(experiment.id)
                 metrics.incr("fetch_jetstream_data.completed")
             else:
                 metrics.incr("fetch_jetstream_data.skipped")
                 logger.info(
                     f"Skipping cache refresh for old experiment {experiment.name}"
+                    + f" ({experiment.slug})"
```

Comment:
f string under f string 👀

Reply:
I think this is correct because it is concatenating, and the second string does not know that the first one is an f-string.

Reply:
I saw that as an option, but wasn't sure what the advantage was over the `+`. Can you elaborate or is it just a style/preference thing?

Reply:
Oh yeah, in general I prefer to avoid using the `+` operator when dealing with strings because it's possible to get into situations sometimes where you might have something like

```py
"this" + something + "that"
```

where you expect `something` to be a string, but it's Python, it could be anything, and then you could either get it cast into a string but formatted in some undesired way, or it could be something that refuses to cast into a string altogether and then you get a runtime error.

So in general it's better to always just wrap everything in an fstring, and then when you've explicitly wrapped everything in fstrings, you can use the multiline paren string join behaviour to make the string as long as you want, and then you don't need the `+` at all, and even though it's redundant and you've already declared everything as a string, you don't need to signal to the reader that there's even a chance something is being cast into a string at all.

So yeah it's mostly a stylistic thing just to make it very explicit that everything is being wrapped in an fstring, nothing is being concatted or cast through the `+` operator, everything is wrapped in an fstring.  Just consistency I guess.

Reply:
Makes sense to me, thanks for the explanation!

Reply:
I wonder if we could enforce this via one of our various linters...

Reply:
There actually is [such a thing](https://github.com/MichaelKim0407/flake8-use-fstring) and we are using it but I don't think it specifically has a rule about preventing `+` concatenation.